www/index.html

```diff
@@ -265,16 +278,36 @@
       </div>
     </div>
 
+    <!-- Templates -->
+    <script type="text/template7" id="notificationTemplate">
+      <li id="{{id}}" {{#unless seen}} class="unseen" {{/unless}}>
+        <a href="#" class="item-link item-content">
+          <div class="item-inner">
+            <div class="item-title-row">
+              <div class="item-title">{{data.title}}</div>
+            </div>
+            <div class="item-subtitle">{{data.body}}</div>
+            <div class="item-text">{{data.extra}}</div>
+            <div class="item-created">Skickades {{js "new Date(this.created_at).toLocaleString()"}}</div>
+          </div>
+        </a>
+      </li>
+    </script>
```

Comment:
Templating is a great way to avoid writing unreadable html in javascript files by the way :) Check out [Template7](http://idangero.us/template7/), which is included in Framework7